components/clamav/files/freshclam.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version='1.0'?>
+<!DOCTYPE service_bundle SYSTEM '/usr/share/lib/xml/dtd/service_bundle.dtd.1'>
+<service_bundle type='manifest' name='clamav'>
+  <service name='antivirus/freshclam' type='service' version='0'>
+    <single_instance/>
+    <dependency name='fs' grouping='require_all' restart_on='none' type='service'>
+      <service_fmri value='svc:/system/filesystem/local'/>
+    </dependency>
+    <dependency name='network' grouping='require_all' restart_on='none' type='service'>
+      <service_fmri value='svc:/milestone/multi-user-server'/>
+    </dependency>
+    <dependency name='cron' grouping='require_all' restart_on='none' type='service'>
+      <service_fmri value='svc:/system/cron:default'/>
+    </dependency>
+    <instance name='default' enabled='false'>
+      <exec_method type='method'
+                   name='start'
+                   exec='/usr/bin/freshclam.sh ; /usr/bin/freshclam.sh --add-cron "%{freshclam/cron-schedule}"'
```

Comment:
it seems, something is wrong:
/usr/bin/freshclam.sh --add-cron "0,15,30,45\ *\ *\ *\ *" doesn't work here:
$ pfexec svcadm enable freshclam
$ pfexec crontab -l clamav
crontab: can't open your crontab file.